src/site/apt/development_contribution.apt

```diff
@@ -96,11 +100,26 @@ Contributing Code
       At least one other member of the community should review the change.
       If there is any feedback, address this and repeat the posting process.
 
+    * Update your Pull Request.
+
+      Update your change to address any comments from reviewers.
+
+    * Prepare your change for merge.
+
+      Squash your changes into a single commit with a nice commit message.
```

Comment:
It would be really nice if we had a consistent example with command lines to go along with this description - especially for the rebase/squash :)